opr/recorder.go

```diff
@@ -28,9 +35,7 @@ func NewChainRecorder(con *config.Config, filpath string) (*ChainRecorder, error
 	return c, nil
 }
 
-// WriteMinerCSV will write all the miner related stats for a given chain to a csv. This includes
-// number of records, difficulties hit, and more.
-func (c *ChainRecorder) WriteMinerCSV() error {
+func (c *ChainRecorder) WritePriceCSV(db database.IDatabase, height int64) error {
```

Comment:
initlx() can be removed from this function since it's not specifically used by that function. quickgrader inits it